src/waveform/renderers/waveformsignalcolors.cpp

```diff
@@ -55,6 +55,12 @@ bool WaveformSignalColors::setup(const QDomNode &node, const SkinContext& contex
         m_playPosColor = m_axesColor;
     }
 
+    m_playedOverlayColor = rgbaColorFromString(context.selectString(node, "PlayedOverlayColor"));
+    if (!m_playedOverlayColor.isValid()) {
+        m_playedOverlayColor = QColor(0,0,0,200);
```

Comment:
Please add a comment what this color is.
This should also become a const in th anonymous namesace on to of this file.